sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -239,16 +239,11 @@ public void setFloat(int ordinal, float value) {
     PlatformDependent.UNSAFE.putFloat(baseObject, getFieldOffset(ordinal), value);
   }
 
-  @Override
-  public int size() {
-    return numFields;
-  }
-
   /**
    * Returns the object for column `i`, which should not be primitive type.
    */
   @Override
-  public Object get(int i) {
+  public Object apply(int i) {
```

Comment:
For `Row`, we cannot remove `apply` and `size`, they are public API. For InternalRow, we can  ban them (raise exception).